src/redismodule.h

```diff
@@ -309,6 +309,9 @@ typedef uint64_t RedisModuleTimerID;
  * Use RedisModule_GetModuleOptionsAll instead. */
 #define _REDISMODULE_OPTIONS_FLAGS_NEXT (1<<4)
 
+/* default set of items to flush in an execution unit */
+#define REDISMODULE_FLUSH_FLAG_DEFAULT (1<<0)
```

Comment:
"FLUSH" is too generic. how about 
REDISMODULE_FLUSH_EXEC_UNIT_FLAG_DEFAULT
or
REDISMODULE_EXECUTION_UNIT_FLAG_DEFAULT
i.e. maybe we'll use the later for other execution unit APIs in the future.